File: baekjoon/baekjoon/4435.c
#include <stdio.h>

int main() {
	int c;
	scanf("%d", &c);

	for (int i = 1; i <= c; i++) {
		size_t a, b, c, d, e, f, g;
		scanf("%u %u %u %u %u %u", &a, &b, &c, &d, &e, &f);
		size_t sum1 = a * 1 + b * 2 + c * 3 + d * 3 + e * 4 + f * 10;
		scanf("%u %u %u %u %u %u %u", &a, &b, &c, &d, &e, &f, &g);
		size_t sum2 = a * 1 + b * 2 + c * 2 + d * 2 + e * 3 + f * 5 + g * 10;

		if (sum2 == sum1) {
			printf("Battle %d: No victor on this battle field\n", i);
		}
		else if (sum1 < sum2) {
			printf("Battle %d: Evil eradicates all trace of Good\n", i);
		}
		else {
			printf("Battle %d: Good triumphs over Evil\n", i);
		}
	}
}

File: baekjoon/baekjoon/4435/4435.java
import java.util.Scanner;


class Main {
	static Scanner s = new Scanner(System.in);

	public static void main(String[] args) {
		int t = s.nextInt();
		for (int i = 0; i < t; i++) {
			battle((i + 1));
		}
	}

	static void battle(int n) {
		long scoregood = 0;
		long scorebad = 0;
		for (int i = 0; i < 6; i++) {
			switch (i) {
				case 0:
					scoregood += s.nextLong();
					break;
				case 1:
					scoregood += s.nextLong() * 2;
					break;
				case 2:
				case 3:
					scoregood += s.nextLong() * 3;
					break;
				case 4:
					scoregood += s.nextLong() * 4;
					break;
				case 5:
					scoregood += s.nextLong() * 10;
					break;
			}
		}
		for (int i = 0; i < 7; i++) {
			switch (i) {
				case 0:
					scorebad += s.nextLong();
					break;
				case 1:
				case 2:
				case 3:
					scorebad += s.nextLong() * 2;
					break;
				case 4:
					scorebad += s.nextLong() * 3;
					break;
				case 5:
					scorebad += s.nextLong() * 5;
					break;
				case 6:
					scorebad += s.nextLong() * 10;
					break;
			}
		}
		System.out.print("Battle " + n + ": ");
		if (scoregood > scorebad) System.out.println("Good triumphs over Evil");
		else if (scoregood < scorebad) System.out.println("Evil eradicates all trace of Good");
		else System.out.println("No victor on this battle field");
	}
}